liz-wheeler.html

```diff
@@ -0,0 +1,16 @@
+
+<link rel="stylesheet" href="liz-wheeler.css">
```

Comment:
I think there may have been a misunderstanding. You did a good job creating the nav element but according to the directions in the README.md you needed to create this in the index.html file, at the top of the file. the fact you created it shows you understand html but need to be mindful of the directions in the README